rhtconsulting/rhc-ose
# Full install

#### What does this PR do?

This is the early attempts at stringing together the other CASL roles to automate an OSE installation on Openstack.
#### How should this be manually tested?

Configure the inventory file similar to  rhc-ose-ansible/inventory/ose-provision and make sure you fill out your rhn username and password fields, as well as a valid subscription manager pool_id.

Spin up the development environment image.
`rhc-ose/docker/openstack-docker-client/run.sh --repository=<rhc-ose>`

Run the ose-provision.yml playbook
`ansible-playbook -i inventory_file rhc-ose/rhc-ose-ansible/ose-provision.yml`
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @oybed  @etsauer @sabre1041
